nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Cli.Test/EntryPointTests.Update.cs

```diff
@@ -353,7 +353,6 @@ await File.WriteAllTextAsync(projectPath, """
                 """);
             var executableName = Path.Join(Path.GetDirectoryName(GetType().Assembly.Location), "NuGetUpdater.Cli.dll");
             var executableArgs = string.Join(" ",
```

Comment:
This change is to resolve a compiler ambiguity with the addition of a `params ReadOnlySpan<T>` overload added in .NET 9.

Reply:
This line will conflict with #10676 which is getting merged soon.